config/job-config.yaml

```diff
@@ -256,6 +268,12 @@ matrix:
         variables:
           - name: docNum
             value: 10000000
+          - name: concurrencyMechanism
```

Comment:
Ok so this approach will certainly work.  But since Python will fall if this is not sent, it's going to be a 6-line chunk that's going to have to be C&Ped into every workload.  Could you check if it works if instead set in the settings.variables section at the bottom of this file?  That should get merged into every workoad.

Reply:
Otherwise, this review looks great.

Reply:
It works, I've moved it there